package.json

```diff
@@ -7,7 +7,10 @@
   "module": "src/index.js",
   "license": "MIT",
   "repository": "hyperapp/hyperapp",
-  "files": ["src", "dist"],
+  "files": [
```

Comment:
It determines which files & folders get published to `npm`.
https://docs.npmjs.com/files/package.json#files